Buildings/Examples/VAVReheat/Controls/SupplyAirTemperatureSetpoint.mo

```diff
@@ -0,0 +1,87 @@
+within Buildings.Examples.VAVReheat.Controls;
```

Comment:
Again I see a few usages `set point`. While I do not mind or have preference, I do think this should be harmonized within the library.

Reply:
This will be addressed through #2225